sdks/how-to/how-to-manage-healthcare-elements.md

```diff
@@ -4,3 +4,283 @@ description: Learn how to manage healthcare elements
+to them.
+
+:::
+
+
+
+## Sharing a Healthcare Element with a Patient
+
+After creating the Healthcare Element, the Healthcare Professional shares it with the Patient.
+
+<!-- file://code-samples/how-to/manage-healthcare-elements/index.mts snippet:HE sharing with data owner-->
+```typescript
+const sharedHealthcareElement = await api.healthcareElementApi.giveAccessTo(
+  healthcareElement,
+  patient.id,
+)
+```
+
+If the operation is successful, the method returns a Promise with the updated Healthcare Element.
+
+## Retrieving a Healthcare Element Using its ID
+
+A single Healthcare Element can be retrieved from the Backend using its id.
+
+<!-- file://code-samples/how-to/manage-healthcare-elements/index.mts snippet:retrieve a HE as data owner-->
+```typescript
+const healthcareElementToRetrieve = new HealthcareElement({
+  description: 'To retrieve, I must create',
+})
+
+const createdHealthcareElement = await api.healthcareElementApi.createOrModifyHealthcareElement(
+  healthcareElementToRetrieve,
+  patient.id,
+)
+
+const retrievedHealthcareElement = await api.healthcareElementApi.getHealthcareElement(
+  createdHealthcareElement.id,
+)
+```
+
+:::caution
+
+Trying to retrieve a Healthcare Element you do not have access to will produce an error.
+
+:::
+
+## Modifying a Healthcare Element
+
+Given an existing Healthcare Element, it is possible to modify it.  
+Some fields, like id and rev, cannot be modified.
+
+<!-- file://code-samples/how-to/manage-healthcare-elements/index.mts snippet:modify a HE as data owner-->
+```typescript
+const yetAnotherHealthcareElement = await api.healthcareElementApi.createOrModifyHealthcareElement(
+  new HealthcareElement({
+    description: 'To modify, I must create',
+  }),
+  patient.id,
+)
+
+const modifiedHealthcareElement = {
+  ...yetAnotherHealthcareElement,
+  description: 'I can change and I can add',
+  openingDate: new Date('2019-10-12').getTime(),
+}
+
+const modificationResult = await api.healthcareElementApi.createOrModifyHealthcareElement(
+  modifiedHealthcareElement,
+  patient.id,
+)
+```
+
+If the operation is successful, the method returns the updated Healthcare Element as it is stored in the Backend.
+
+:::caution
+
+To update a Healthcare Element, both id and rev fields must be valid.
+
+:::
+
+## Retrieving Healthcare Element Using Complex Search Criteria
+
+If you want to retrieve a set of Healthcare Element that satisfy complex criteria, you can use a Filter.  
+In the following example, you will instantiate a filter to retrieve all the Healthcare Element of a Patient that a Healthcare Professional
+ can access
+
+<!-- file://code-samples/how-to/manage-healthcare-elements/index.mts snippet:create HE filter-->
+```typescript
+const healthcareElementFilter = await new HealthcareElementFilter()
+  .forDataOwner(user.healthcarePartyId)
+  .forPatients(api.cryptoApi, [patient])
+  .build()
+```
+
+:::note
+
+You can learn more about filters in the how to.
```

Comment:
I think there's already the empty file, so that you can add the link, even if it is empty for now